draft-bradleylundberg-cfrg-arkg.md

```diff
@@ -512,24 +513,20 @@ Then the `BL` parameter of ARKG may be instantiated as follows:
 - Elliptic curve scalar values are encoded to and from octet strings
   using the procedures defined in sections 2.3.7 and 2.3.8 of [SEC1].
 
```

Comment:
Add bullet point (harmonized with [RFC 9380](https://www.rfc-editor.org/rfc/rfc9380.html)):

- `namedCurve` is a set of functions that return parameters relevant for suites that rely on elliptic curves. Parameters relevant for to ARKG are:
    - `F,q,p` that defines a finite field `F` of characteristic `p` and `#F = q = p^m` where `m` is an integer. For prime fields, `q = p`.
    - `E` is an elliptic curve specified by an equation and a field `F`.
    - `n` is the order of `E`.
    - `G` is a prime-order subgroup of the points on `E`.
    - `r` is a prime factor of `n` and the order of `G`.
    - `P` is a generator element of `G`.